src/session.rs

```diff
@@ -374,13 +378,27 @@ impl Session {
             let stream_mid_rid = self.streams.stream_rx_by_mid_rid(mid, Some(rid));
 
             let ssrc_main = if is_repair {
-                stream_mid_rid?.ssrc()
+                let Some(stream) = stream_mid_rid else {
+                    return None;
+                };
```

Comment:
Why not keep the `?`

Reply:
Because I didn't need just ssrc anymore so it felt cleaner to check once if the stream existed but it could just be

```
let stream = stream_mid_rid?;
```